src/staticanalysis/bot/static_analysis_bot/tasks/coverage.py

```diff
@@ -75,7 +73,6 @@ def as_dict(self):
             'path': self.path,
             'line': self.line,
             'message': self.message,
-            'is_third_party': self.is_third_party(),
```

Comment:
We likely need to keep the is_third_party check for the coverage case

Reply:
But you can remove the is_third_party function from the base class and just put it in this class

Reply:
Or maybe not, I see we were not using the info before

Reply:
restored in 72cf2d4adee8ff6310b89f7bf34df9cb97518875